.github/CODEOWNERS

```diff
@@ -21,6 +21,8 @@
 /sdk/core/logger @xirzec
 
 # Service teams
+/sdk/appconfiguration/ @richardpark-msft
+
```

Comment:
nit: extra line

Reply:
@danieljurek: Are you suggesting to add an extra line or remove the one I added?  I added the empty line because it appears these are organized in "groups" with empty lines between them.

Reply:
_expands the file farther down..._ let's keep the extra line... it was other CODEOWNERS files (python and java) that have the services bunched together.